packages/driver/cypress/e2e/issues/28527.cy.ts

```diff
@@ -0,0 +1,31 @@
+describe('issue 28527', { testIsolation: false, retries: 2 }, () => {
+  before(() => {
+    cy.visit('/fixtures/empty.html')
+  })
+
+  beforeEach(() => {
+    // ensure we run the tests in run mode
+    // @ts-expect-error
+    Cypress.config('isInteractive', false)
+    // @ts-expect-error
+    Cypress.config('exit', true)
+  })
+
+  // there can only be one test in this file to ensure we are testing the scenario
+  // where a test fails and the runner does not navigate to about:blank between retries
+  it('test 1', () => {
```

Comment:
can we give this a more descriptive name to what we are testing?

Reply:
updated [60870f8](https://github.com/cypress-io/cypress/pull/30864/commits/60870f8dfdff7db31f36ef7211323cf9204adbd5)